src/components/Editor/Preview.css

```diff
@@ -28,7 +28,7 @@
 }
 
 .selected-token {
-  background-color: var(--theme-search-overlays-semitransparent);
+  background-color: rgb(255, 255, 180);
   color: var(--theme-selection-color);
```

Comment:
@ofiron01 You could look in here to see if we have a variable matching the colour we want, and use that.  https://github.com/devtools-html/devtools-core/blob/master/packages/devtools-launchpad/src/lib/themes/variables.css#L1

If not, we can do a PR in devtools-core to add a new variable.